galaxy/autoupgrade_db.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+do_backup () {
+   backupdest="/export/preupgradebackup_"$(date +%Y_%m_%d_%H_%M_%S)
+   mkdir -p "${backupdest}" && cp -R /export/.distribution_config /export/galaxy-central /export/postgresql  /export/shed_tools /export/var "${backupdest}"
```

Comment:
.distribution_config does not need to be copied, not sure about the rest, I think it's enough to backup the database is this mode.